crates/examples/src/readobj/elf.rs

```diff
@@ -2940,6 +2950,58 @@ static FLAGS_R_NDS32: &[Flag<u32>] = &flags!(
     R_NDS32_TLS_TPOFF,
     R_NDS32_TLS_DESC,
 );
+static FLAGS_R_LOONGARCH: &[Flag<u32>] = &flags!(
+    R_LARCH_NONE,
+    R_LARCH_32,
+    R_LARCH_64,
+    R_LARCH_RELATIVE,
+    R_LARCH_COPY,
+    R_LARCH_JUMP_SLOT,
+    R_LARCH_TLS_DTPMOD32,
+    R_LARCH_TLS_DTPMOD64,
+    R_LARCH_TLS_DTPREL32,
+    R_LARCH_TLS_DTPREL64,
+    R_LARCH_TLS_TPREL32,
+    R_LARCH_TLS_TPREL64,
+    R_LARCH_IRELATIVE,
+    R_LARCH_MARK_LA,
+    R_LARCH_MARK_PCREL,
+    R_LARCH_SOP_PUSH_PCREL,
+    R_LARCH_SOP_PUSH_ABSOLUTE,
+    R_LARCH_SOP_PUSH_DUP,
+    R_LARCH_SOP_PUSH_GPREL,
+    R_LARCH_SOP_PUSH_TLS_TPREL,
+    R_LARCH_SOP_PUSH_TLS_GOT,
+    R_LARCH_SOP_PUSH_TLS_GD,
+    R_LARCH_SOP_PUSH_PLT_PCREL,
+    R_LARCH_SOP_ASSERT,
+    R_LARCH_SOP_NOT,
+    R_LARCH_SOP_SUB,
+    R_LARCH_SOP_SL,
+    R_LARCH_SOP_SR,
+    R_LARCH_SOP_ADD,
+    R_LARCH_SOP_AND,
+    R_LARCH_SOP_IF_ELSE,
+    R_LARCH_SOP_POP_32_S_10_5,
+    R_LARCH_SOP_POP_32_U_10_12,
+    R_LARCH_SOP_POP_32_S_10_12,
+    R_LARCH_SOP_POP_32_S_10_16,
+    R_LARCH_SOP_POP_32_S_10_16_S2,
+    R_LARCH_SOP_POP_32_S_5_20,
+    R_LARCH_SOP_POP_32_S_0_5_10_16_S2,
+    R_LARCH_SOP_POP_32_S_0_10_10_16_S2,
+    R_LARCH_SOP_POP_32_U,
+    R_LARCH_ADD8,
+    R_LARCH_ADD16,
+    R_LARCH_ADD24,
+    R_LARCH_ADD32,
+    R_LARCH_ADD64,
+    R_LARCH_SUB8,
+    R_LARCH_SUB16,
+    R_LARCH_SUB24,
+    R_LARCH_SUB32,
+    R_LARCH_SUB64,
```

Comment:
Is this missing `R_LARCH_GNU_VTINHERIT` and `R_LARCH_GNU_VTENTRY`?

Reply:
Yes, I have added these two constants.